ush/load_fv3gfs_modules.sh

```diff
@@ -24,10 +24,10 @@ elif [[ -d /lfs3 ]] ; then
 	module load module_base.jet 
 elif [[ -d /scratch1 ]] ; then
 	# We are on NOAA Hera
-	module load module_base.hera
+	module use "$HOMEgfs/sorc/ufs_coupled.fd/modulefiles"
+	module load ufs_hera.intel
 	if [[ -d $HOMEgfs/sorc/ufs_coupled.fd/GOCART ]] ; then
-		module use "$HOMEgfs/sorc/ufs_coupled.fd/modulefiles/hera.intel"
-		module load fv3
+		module load ufs_aerosols_hera.intel
```

Comment:
This modulefile should go away soon.

Reply:
The issue is the if the GOCART directory exists is not going to work once the ufs-weather-model PR is committed and all scenarios have the folder GOCART.  So this will have to not adversely affect the other scenarios.

Reply:
Agreed. We should limit use of the additional aerosol modulefile only to coupled configurations building GOCART.

Reply:
See comment above.

Reply:
Work is ongoing to include all GOCART dependencies into hpc-stack.